media-source-respec.html

```diff
@@ -2706,6 +2714,367 @@ <h3>Event Summary</h3>
       </section>
     </section>
 
+    <section data-cite="mimesniff" data-dfn-for="ManagedMediaSource">
+      <h2><dfn>ManagedMediaSource</dfn> interface</h2>
+      <p>
+        A {{ManagedMediaSource}} is a {{MediaSource}} that actively manages its
+        memory content. Unlike a {{MediaSource}}, the [=user agent=] MAY evict it
```

Comment:
It's not quite clear what the "it" here is referring to, so I'd suggest a phasing such as:

> the user agent MAY evict data from a ManagedSourceBuffer when under memory pressure

Reply:
I re-wrote it as:
```
        A {{ManagedMediaSource}} is a {{MediaSource}} that actively manages its
        memory content. Unlike a {{MediaSource}}, the [=user agent=] MAY evict evict content
        from its {{MediaSource/sourceBuffers}} (populated with {{ManagedSourceBuffer}}) for any reasons, including when under [=ManagedMediaSource/memory
        pressure=].

```

Reply:
One minor nit, "reasons" should be "reason", but otherwise this looks good to me